tripal_chado/src/Plugin/TripalImporter/FASTAImporter.php

```diff
@@ -43,16 +44,21 @@ public function form($form, &$form_state) {
     // Always call the parent form to ensure Chado is handled properly.
     $form = parent::form($form, $form_state);
 
-    // get the list of organisms
-    $organisms = chado_get_organism_select_options();
+    $settings = \Drupal::config('tripal.settings');
 
-    $form['organism_id'] = [
-      '#title' => t('Organism'),
-      '#type' => 'select',
-      '#description' => t("Choose the organism to which these sequences are associated"),
-      '#required' => TRUE,
-      '#options' => $organisms,
-    ];
+    $options = [];
+    // Set some defaults to keep each of the fields simpler.
+    $options['select_limit'] = $settings->get('tripal_entity_type.widget_global_select_limit') ?? 50;
+    $options['match_limit'] = $settings->get('tripal_entity_type.match_limit') ?? 10;
+    $options['match_operator'] = $settings->get('tripal_entity_type.match_operator') ?? 'CONTAINS';
+    $options['size'] = $settings->get('tripal_entity_type.size');
+    $options['placeholder'] = $settings->get('tripal_entity_type.placeholder');
```

Comment:
It would be great to move these into the controller as defaults so that not everyone has to write this code ;-p I would add these at the top of all the form element methods for if the caller did not supply them. That way these are all optional!